src/main/java/dk/cngroup/lentils/entity/FinalPlace.java

```diff
@@ -0,0 +1,92 @@
+package dk.cngroup.lentils.entity;
+
+import org.springframework.data.geo.Point;
+
+import javax.persistence.*;
+import javax.validation.Constraint;
+import javax.validation.constraints.Size;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+@Table(name = "finalPlace")
```

Comment:
Není třeba dodržovat konvenci spíše s podtržítkama ve jménech db objektů? Malé velké písmena jsou někdy problém.

Reply:
Třeba v případech, kdyby se to pak převádělo na jinou db apod.